packages/clay-css/src/scss/components/_multi-step-nav.scss

```diff
@@ -62,7 +62,9 @@
 			@if ($multi-step-icon-active-bg-image) {
 				&[data-multi-step-icon] {
 					&:before {
-						content: normal;
+						// https://issues.liferay.com/browse/LPS-147457
+
+						content: none;
```

Comment:
Hey @pat270 just out of curiosity, what effect does this have on `:before`? Is this related to some other value?

Reply:
@matuzalemsteles this is exactly the same as `content: normal`. `content: none` gets a pass on the OAW Tracker for some reason 🤔.

From https://developer.mozilla.org/en-US/docs/Web/CSS/content:
> normal
Computes to none for the ::before and ::after pseudo-elements.

For

> we will keep the content but let the developers change it via the sass variable.

This is correct, but I think the person who reported this issue won't have to change anything after these get merged.

Reply:
> this is exactly the same as content: normal. content: none gets a pass on the OAW Tracker for some reason 🤔.

Yeah I also found it strange because it seems that the behavior is the same, maybe it has some other effect that we don't know 🤷‍♂️

Reply:
Thanks @matuzalemsteles, I'm going to make some minor adjustments. Let's just say the tool isn't perfect.